Sources/Workspace/WorkspaceState.swift

```diff
@@ -265,6 +265,9 @@ fileprivate struct WorkspaceStateStorage {
                     switch kind {
                     case "local":
                         self.init(underlying: .local)
+                    case "localArchived":
```

Comment:
I don't see any tests for `WorkspaceState`? Is it because a DiagnosticsEngine is not yet available in this part, or should I introduce ones?

Reply:
It is not needed anymore.